.bingo/golangci-lint.mod

```diff
@@ -1,5 +1,7 @@
 module _ // Auto generated by https://github.com/bwplotka/bingo. DO NOT EDIT
 
-go 1.22.1
+go 1.23.0
 
-require github.com/golangci/golangci-lint v1.62.2 // cmd/golangci-lint
+toolchain go1.23.4
```

Comment:
This toolchain line is not present in the go.mod for golangci-lint 1.64.5. I suspect this is another artifact of your environment. Can you remove this?

https://github.com/golangci/golangci-lint/blob/0a603e49e5e9870f5f9f2035bcbe42cd9620a9d5/go.mod#L3